app/models/ability.rb

```diff
@@ -155,6 +158,8 @@ def signed_in_with_cfp_role(user)
     can :manage, CallForPaper, conference_id: conf_ids_for_cfp
     can :manage, Commercial, commercialable_type: 'Event',
                              commercialable_id: Event.where(conference_id: conf_ids_for_cfp).pluck(:id)
+    can [:read, :create], Comment, commentable_type: 'Event',
```

Comment:
I think we should explicitely define the actions we want, **read** is not a controller action, it is an alias the gem offers but the actions it corresponds to could change on a future gem update.